browser/ui/views/toolbar/brave_toolbar_view.cc

```diff
@@ -325,15 +325,3 @@ void BraveToolbarView::Layout() {
   app_menu_button_->SetBounds(next_element_x, toolbar_button_y, app_menu_width,
                               toolbar_button_height);
 }
-
-gfx::Size BraveToolbarView::GetSizeInternal(
```

Comment:
We're not using Flex layout since we still override `ToolbarView::Layout` (at https://github.com/brave/brave-core/pull/1629/files#diff-3d72382b1c4fba0ded2fc0100530ff54L185). We'll likely need to re-do our custom spacing within the flex layout since child components report their sizing in flex-terms now (e.g. actions / extensions container).

Reply:
@petemill can you take this aspect? I'd like to not block on landing this for that and you can work on that in a follow up.

Reply:
I tested this with 8 extensions, it no longer overflows into the menu but it makes the urlbar smaller. 
For wide mode it makes the URL bar wider, and for non wide mode it leaves some extra space.  Everything seems to work from what I can tell.